src/intentManager.js

```diff
@@ -305,6 +305,18 @@ define([
         }
     });
 
+    function noIntents(features) {
+        return !features.custom_intents && !features.templated_intents;
+    }
+
+    function templatedIntents(features) {
+        return features.templated_intents && !features.custom_intents;
```

Comment:
if `nointents` then there will be no intent question types (community & standard)
if `templatedIntents` then there will be a template for the intent with extra and response key value fields moved to advanced
if custom intents then it will be the same as it always has been

i changed `templatedIntents` to `onlyTemplatedIntents` as that's a little more obvious